build/build-cdt-lib.js

```diff
@@ -221,7 +221,7 @@ function doModification(modification) {
       }
 
       if (removeNode) {
-        return ts.createNode(ts.SyntaxKind.Unknown);
+        return /** @type {ts.Node} */ ({kind: ts.SyntaxKind.Unknown});
```

Comment:
> Did this come from somewhere?

autocomplete-driven API discovery :) I never found much useful reference code using these APIs, so I just found whatever worked through trial and error.

Reply:
Yeah aside from the jsdoc for `substituteNode` I couldn't find any docs or examples for the `substituteNode` option to `ts.createPrinter`. Github search isn't very helpful, though, since you can't filter out TypeScript repo clones in results